requirements.txt

```diff
@@ -1,7 +1,8 @@
 numpy==1.24.3
 opencv-python==4.7.0.72
 onnx==1.14.0
-onnxruntime==1.15.0
```

Comment:
is there a reason behind removing onnxruntime?

People without a GPU will need it.

Reply:
It does not work with both libs installed for some reason, took me hours to find out.